test/livebook_web/live/hub/new/enterprise_component_test.exs

```diff
@@ -80,9 +111,13 @@ defmodule LivebookWeb.Hub.New.EnterpriseComponentTest do
         }
       })
 
-      assert view
-             |> element("#connect")
-             |> render_click() =~ "Add Hub"
+      view
+      |> element("#connect")
+      |> render_click()
+
+      Process.sleep(50)
+
+      assert render(view) =~ "bf1587a3-4501-4729-9f53-43679381e28b"
```

Comment:
Worth extracting into a module variable, mostly to make it cleaner what it is :)

Reply:
I believe this is temporary, we will change it once we have actual users. :)

Reply:
Yeah, it is